components/script/dom/node.rs

```diff
@@ -2017,6 +2017,16 @@ impl Node {
         parent.owner_doc().remove_script_and_layout_blocker();
     }
 
+    // https://dom.spec.whatwg.org/multipage/#string-replace-all
```

Comment:
```suggestion
    // https://dom.spec.whatwg.org/#string-replace-all
```

only the HTML spec uses `multipage`